recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+        # thrift
+        - test -f ${PREFIX}/{{ install_base }}/completion_hints.thrift  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/heavy.thrift  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/common.thrift  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/serialized_result_set.thrift  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/extension_functions.thrift  # [unix]
+        - if not exist %PREFIX%\completion_hints.thrift exit 1  # [win]
+        - if not exist %PREFIX%\heavy.thrift exit 1  # [win]
+        - if not exist %PREFIX%\common.thrift exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine/serialized_result_set.thrift exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine/extension_functions.thrift exit 1  # [win]
+        # includes
+        - test -f ${PREFIX}/{{ install_base }}/Shared/funcannotations.h  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/Shared/InlineNullValues.h  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/Logger/Logger.h  # [unix]
+        - if not exist %PREFIX%\Shared\funcannotations.h exit 1  # [win]
+        - if not exist %PREFIX%\Shared\InlineNullValues.h exit 1  # [win]
+        - if not exist %PREFIX%\Logger\Logger.h exit 1  # [win]
+        # QE
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/heavydbTypes.h  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/RuntimeFunctions.bc  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/GeosRuntime.bc  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/ExtensionFunctions.ast  # [unix]
+        - if not exist %PREFIX%\QueryEngine\heavydbTypes.h exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\RuntimeFunctions.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\GeosRuntime.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\ExtensionFunctions.ast exit 1  # [win]
+        # jar
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavyai-utility-{{ base_version }}.jar  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavyai-jdbc-{{ base_version }}.jar  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/calcite-1.0-SNAPSHOT-jar-with-dependencies.jar  # [unix]
+        - if not exist %PREFIX%\bin\heavyai-utility-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\heavyai-jdbc-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\calcite-1.0-SNAPSHOT-jar-with-dependencies.jar exit 1  # [win]
+        # Unspecified
+        # startheavy and insert_sample_data are bash scripts
+        - test -f ${PREFIX}/{{ install_base }}/bin/startheavy  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavydb_insert_sample_data  # [unix]
+        #
+        - test -f ${PREFIX}/{{ install_base }}/bin/generate_cert  # [unix]
+        - if not exist %PREFIX%\bin\generate_cert exit 1  # [win]
```

Comment:
This test section is quite something 😂  A file called `test_heavydb.{sh,bat}` might be cleaner, but up to you!

Reply:
I can fix this one either in the old one or in this one, once it gets approved.